CMakeLists.txt

```diff
@@ -42,7 +42,8 @@ roslint_cpp()
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-  DEPENDS Eigen
+  INCLUDE_DIRS include
```

Comment:
technically not needed for the Eigen upgrade, but generally a good idea to have this for any dependent projects that may want to include `mapper` code